env_spec.py

```diff
@@ -19,138 +24,158 @@
 def create_list(input_str):
     """
     Creates a list from input_str_list.Returns the new list.
+    If input_str=[1,2,], returns empty list(syntax error).
     """
     input_str = input_str.replace("[", "")
     input_str = input_str.replace("]", "")
     new_list = input_str.split(",")
 
+    if "" in new_list:
+        return []
+
     for pos in range(0, len(new_list)):
-        new_list[pos] = str(new_list[pos])
+        new_list[pos] = str(new_list[pos]).strip()
 
     return new_list
 
 
-def check_input(input_field):
-    """
-    Checks syntax of input field(uppercase latin chars,numeric digits, underscores).
-    Returns False for error or True for correct syntax.
-    """
-    alphanumeric_that_does_not_start_with_digit = r"^[A-Z_][0-9A-Z_]*$"
-
-    ret = re.match(alphanumeric_that_does_not_start_with_digit, input_field)
-
-    if ret is None:
-        return False
-
-    return True
-
-
-def check_type(input_type):
-    """
-    Checks if given type is valid and returns True, else returns False.
-    """
+def render_to_html(env_spec_list):
+    html_output = ""
 
-    if "[" in input_type and "=" in input_type:
-        env_var_type, value = input_type.split("=")
-        value = value.strip()
+    for dict in env_spec_list:
```

Comment:
Please use another variable name than `dict`, as it's a built-in Python type. What about `env_spec_entry`?